core/src/test/java/org/testcontainers/images/builder/DockerfileBuildTest.java

```diff
@@ -0,0 +1,67 @@
+package org.testcontainers.images.builder;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+
+@RunWith(Parameterized.class)
+public class DockerfileBuildTest {
+
+    private static final Path RESOURCE_PATH = Paths.get("src/test/resources/dockerfile-build-test");
+
+    public String expectedFileContent;
+    public ImageFromDockerfile image;
+
+    public DockerfileBuildTest(String expectedFileContent, ImageFromDockerfile image) {
+        this.expectedFileContent = expectedFileContent;
+        this.image = image;
+    }
+
+    @Parameterized.Parameters
```

Comment:
I would move the static method before the instance methods/fields (I think we do it in other places), but that's up to you :)

Reply:
Good point - done